## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :x: [more...](#3f1e3fba-6fa6-41e7-829d-f04aff06da99) |
<a name="3f1e3fba-6fa6-41e7-829d-f04aff06da99"></a>
### 8260-49eaa4a/mint-gateway-nas.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.54:31048
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 0c83daf8bb63:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 0 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 41 seconds
(4/14) Running aws-sdk-ruby tests ... done in 2 seconds
(5/14) Running awscli tests ... done in 1 minutes and 2 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 29 seconds
(8/14) Running minio-dotnet tests ... done in 25 seconds
(9/14) Running minio-go tests ... done in 55 seconds
(10/14) Running minio-java tests ... done in 14 seconds
(11/14) Running minio-js tests ... done in 39 seconds
(12/14) Running minio-py tests ... FAILED in 1 minutes and 2 seconds
{
  "duration": 5021,
  "status": "FAIL",
  "name": "minio-py:test_presigned_put_object_expiry_5sec",
  "error": "Traceback (most recent call last):\n  File \"/mint/run/core/minio-py/tests.py\", line 1489, in test_presigned_put_object_expiry_5sec\n    raise ValueError('Presigned put url failed to expire!')\nValueError: Presigned put url failed to expire!\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/mint/run/core/minio-py/tests.py\", line 2045, in main\n    test_presigned_put_object_expiry_5sec(client, log_output)\n  File \"/mint/run/core/minio-py/tests.py\", line 1491, in test_presigned_put_object_expiry_5sec\n    raise Exception(err)\nException: Presigned put url failed to expire!\n",
  "function": "presigned_put_object(bucket_name, object_name, expires)",
  "args": {
    "object_name": "668a784f-02b1-4a13-b28d-a9543def0862",
    "bucket_name": "minio-py-test-743214e6-1d56-44bd-97cf-c594f0e2eecf"
  },
  "message": "Presigned put url failed to expire!"
}

Executed 11 out of 14 tests successfully.
```

--- a/docs/sts/ldap.md
+++ b/docs/sts/ldap.md
@@ -92,7 +92,7 @@ only.
 
 ## Notes on configuring with Microsoft Active Directory (AD)
 
-The LDAP STS API also works with Microsoft AD and can be configured in a manner
+The LDAP STS API also works with Microsoft AD and can be configured
 as above. The following are some notes on determining the values of
 the configuration parameters described above.
 

--- a/docs/sts/ldap.md
+++ b/docs/sts/ldap.md
@@ -93,7 +93,7 @@ only.
 ## Notes on configuring with Microsoft Active Directory (AD)
 
 The LDAP STS API also works with Microsoft AD and can be configured in a manner
-similar to the above. The following are some notes on determining the values of
+as above. The following are some notes on determining the values of
 the configuration parameters described above.
 
 Once LDAP over TLS is enabled on AD, test access to LDAP works by running a

